vite.config.ts

```diff
@@ -10,9 +10,15 @@ export default defineConfig({
     setupFiles: ["test-setup.ts"],
     coverage: {
       provider: "istanbul",
-      reporter: ["cobertura"],
+      reporter: process.env.GITHUB_ACTIONS
+        ? ["github-actions", "cobertura"]
+        : ["text", "html", "clover", "json"],
+      reportsDirectory: "html/coverage",
```

Comment:
Why do we need HTML reports at all if we are using `codecov`s app?

Reply:
How else would you view coverage locally without pushing changes first?